service/upstart/upstart.go

```diff
@@ -349,8 +349,8 @@ script
 {{if .Logfile}}
   # Ensure log files are properly protected
   touch {{.Logfile}}
-  chown syslog:syslog {{.Logfile}}
-  chmod 0600 {{.Logfile}}
+  chown syslog:adm {{.Logfile}}
+  chmod 0640 {{.Logfile}}
```

Comment:
@howbazaar I did not touch it, as upstart is not used in newer Ubuntu version and it would make for a bigger change extracting the template and placing the core/paths values.

Reply:
That's fine.